### PR TITLE
Initial mpMetrics-2.1 implementation

### PR DESCRIPTION
**Pull request by Yushan-Lin**
#9068

### CODE REVIEW

**Comment by Yushan-Lin** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/mpMetrics-2.1/com.ibm.websphere.appserver.mpMetrics-2.1.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ com.ibm.websphere.appserver.restHandler-1.0, \
+ com.ibm.websphere.appserver.servlet-3.1; ibm.tolerates:=4.0,\
+ com.ibm.websphere.appserver.mpConfig-1.3; ibm.tolerates:=1.4
+-bundles=com.ibm.ws.require.java8, \
+ com.ibm.ws.microprofile.metrics.common, \
+ com.ibm.ws.microprofile.metrics.2.0, \
+ com.ibm.ws.microprofile.metrics.2.1, \
+ com.ibm.ws.microprofile.metrics.private.2.1, \
+ com.ibm.ws.microprofile.metrics.public.2.1
+kind=ga
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/mpMetrics-2.1/com.ibm.websphere.appserver.mpMetrics-2.1.feature]: please initially deliver as noship, move to beta for first release and then only move to ga once all focal point approvals have been obtained.

Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/public/mpMetrics-2.1/com.ibm.websphere.appserver.mpMetrics-2.1.feature]: Also, are the FAT test mostly duplicated from the previous versions? I suggest that you make use of the RepeatTests rule to re-run the old buckets against the new feature versions.

**Comment by dmuelle** [comment]
Hi, do the English nlsprops files require ID review?

**Comment by Yushan-Lin** [comment]
@dmuelle The nlsprops files were duplicated from other projects.

**Comment by tevans78** [comment]
Best practice is to initially deliver a new feature as noship ... but it isn't madatory so I have approved that part for now. You still need an SME review before delivery.

**Comment by tevans78** [comment]
Similarly for duplicating test code ... it would be better to just re-run the existing tests but I'm not going to block delivery for that.

**Pull request merged**